src/localplayer.cpp

```diff
@@ -663,10 +664,26 @@ void LocalPlayer::applyControl(float dtime, Environment *env)
 	const INodeDefManager *nodemgr = env->getGameDef()->ndef();
 	Map *map = &env->getMap();
 	const ContentFeatures &f = nodemgr->get(map->getNodeNoEx(getStandingNodePos()));
-	bool slippery = (itemgroup_get(f.groups, "slippery") != 0);
+	float slip_factor = 1.0f;
+	int slippery = itemgroup_get(f.groups, "slippery");
+	// REMOVE: can override via env variable
+	char *slipenv = getenv("SLIPPERY");
+	if (slipenv)
+		slippery = atoi(slipenv);
```

Comment:
Oh, that part's not permanent. I didn't even ensure the right headers. This is just to jump-start testing and experimentation.